config/locales/client.en.yml

```diff
@@ -3492,6 +3492,7 @@ en:
 
         embed_by_username: "Username for topic creation"
         embed_post_limit: "Maximum number of posts to embed"
+        embed_link_to_top: "Link to the top of the page"
```

Comment:
I would suggest a description that indicates what the setting does: "When embedding, the continue button links to the top of the topic instead of the bottom"